common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -112,11 +118,17 @@ prplmesh_agent_stop() {
     killall_program beerocks_agent
 }
 
+prplmesh_delete_logs() {
+    echo "deleting logs"
+    rm -rf /tmp/beerocks/logs
+    rm -rf /tmp/$SUDO_USER/beerocks/logs
```

Comment:
SUDO_USER is not reliable. Within the container, you have only USER. Also when running as root natively, you only have USER. So instead, use `${SUDO_USER:-${USER}}`.